src/get-dart/index.md

```diff
@@ -78,33 +78,47 @@ Dart supports the following macOS versions as of November 2023:
 
 The Dart SDK has three release channels:
 
-* **Stable** channel: **stable releases**, updated roughly every three months;
-  currently `[calculating]`{:.build-rev-stable}.
-  
-  Stable releases are suitable for production use.
-  
-* **Beta** channel: **preview releases**, usually updated every month;
-  currently `[calculating]`{:.build-rev-beta}.
-  
-  Beta channel builds are preview builds for the stable channel. We recommend
-  testing, but not releasing, your apps against beta to preview new features or
-  test compatibility with future releases.
-  
-* **Dev** channel: **prereleases**, usually updated twice a week;
-  currently `[calculating]`{:.build-rev-dev}.
-  
-  Dev channel releases are the most current with latest changes, may be broken,
-  are unsupported, and may contain unvetted breaking changes.
-
-**Stable** channel releases of the Dart SDK have `x.y.z` version strings like
-`1.24.3` and `2.1.0`. They consist of dot-separated integers, with no hyphens or
-letters, where `x` is the major version, `y` is the minor version, and `z` is
-the patch version.
-
-**Beta** and **dev** channel releases of the Dart SDK (non-stable releases) have
-`x.y.z-a.b.<beta|dev>` versions like `2.8.0-20.11.beta`. The part before the
-hyphen follows the stable version scheme, `a` and `b` after the hyphen are the
-prerelease and prerelease patch versions, and `beta` or `dev` is the channel.
+### Stable channel
+Dart deploys a release to the _stable_ channel about every three months.
+The current stable version is  `[calculating]`{:.build-rev-stable}.
+
+Use stable channel releases for production environments.
+
+Versioning for releases in stable channel of the Dart SDK follows a `x.y.z` format.
```

Comment:
This paragraph is already covered elsewhere: https://dart.dev/tools/pub/versioning#semantic-versions